README.md

```diff
@@ -73,6 +73,14 @@ localStorage.setItem('myFirstKey', 'myFirstValue');
 console.log(localStorage.getItem('myFirstKey'));
 ```
 
+### Node ###
+
+Preload `localStorage` at node startup as
```

Comment:
... and this to "Polyfil your node.js environment with this as the global localStorage when launching your own code"

Reply:
```suggestion
Polyfil your node.js environment with this as the global localStorage when launching your own code
```